compose-v2/galaxy-bioblend-test/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Waiting for Galaxy..."
+until [ "$(curl -s -o /dev/null -w '%{http_code}' nginx)" -eq "200" ] && echo Galaxy started; do
```

Comment:
ephemeris has a galaxy-wait command for this: https://ephemeris.readthedocs.io/en/latest/commands/galaxy-wait.html

Ephemeris seems to be useful in multiple cases to maybe we should include it?